tests/test_routes.py

```diff
@@ -0,0 +1,61 @@
+from app.models.planet import Planet
+
+def test_get_all_planets_with_no_records(client):
+    # Act
+    response = client.get("/planet")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_one_planet_returns_planet(client, one_saved_planet):
+    response = client.get("/planet/1")
+    response_body = response.get_json()
+
+    # assert
+    assert response.status_code == 200
+    assert response_body["id"] == 1
+    assert response_body["name"] == "Pluto"
+    assert response_body["description"] == "cold and icy"
+    assert response_body["distance"] == 15
```

Comment:
Since you passed in the fixture `one_saved_planet` then your assertions could use the object to access the attribute values like: 
`assert response_body["id"] == one_saved_planet.id`